libse/Dictionaries/OcrFixReplaceList.cs

```diff
@@ -113,14 +113,19 @@ public static OcrFixReplaceList FromLanguageId(string languageId)
                 XmlNode node = doc.DocumentElement.SelectSingleNode(name);
                 if (node != null)
                 {
+                    const string ReplaceWith = "replaceWith";
+                    const string Find = "find";
                     foreach (XmlNode item in node.ChildNodes)
                     {
-                        if (item.Attributes != null && item.Attributes["replaceWith"] != null && item.Attributes["find"] != null)
+                        if (item.Attributes != null && item.Attributes[ReplaceWith] != null && item.Attributes[Find] != null)
                         {
                             string to = item.Attributes["replaceWith"].InnerText;
                             string from = item.Attributes["find"].InnerText;
-                            if (!list.ContainsKey(from))
-                                list.Add(from, to);
+                            if (Utilities.IsValidRegex(to) && Utilities.IsValidRegex(from))
+                            {
+                                if (!list.ContainsKey(from))
+                                    list.Add(from, to);
+                            }
```

Comment:
`to` contains a replacement pattern (see [Substitution Elements and Replacement Patterns](https://msdn.microsoft.com/en-us/library/ewy2t5e0(v=vs.110).aspx)).